moc_sprint_tools/create_sprint_boards.py

```diff
@@ -0,0 +1,106 @@
+import click
+import csv
+import datetime
+import github
+import logging
+
+from moc_sprint_tools.sprintman import BoardNotFoundError
+
+LOG = logging.getLogger(__name__)
+
+CARD_COPYING_MAP = {
+    'notes': 'notes',
+    'backlog': 'backlog',
+    'in progress': 'in progress'
+}
+
+
+def load_sprint_data(input_file):
+    sprints = []
+    with open(input_file) as f:
+        reader = csv.reader(f)
+        next(f)
+        for row in reader:
+            sprints.append(
+                [row[0], datetime.datetime.strptime(row[1], '%Y-%m-%d')])
+    return sprints
+
+
+def get_column(board, name):
+    for c in board.get_columns():
+        if c.name.lower() == name.lower():
+            return c
+
+
+def copy_card(source_card, destination_column):
+    content = source_card.get_content()
+    if content:
+        if isinstance(content, github.Issue.Issue):
+            content_type = 'Issue'
+        elif isinstance(content, github.PullRequest.PullRequest):
+            content_type = 'PullRequest'
+        else:
+            LOG.warning('Couldn\'t copy card with unkown type')
+            return
+
+        LOG.info('adding card "%s" to %s', (content.title,
+                                            destination_column.name))
+        destination_column.create_card(
+            content_id=content.id,
+            content_type=content_type,
+        )
+    else:
+        destination_column.create_card(note=source_card.note)
+
+
+def copy_board(source_board, destination_board):
+    for source in source_board.get_columns():
+        destination = CARD_COPYING_MAP.get(source.name.lower(), None)
+        if not destination:
+            continue
+
+        destination = get_column(destination_board, destination)
+        cards = list(source.get_cards())
+        cards.reverse()  # Creating a card adds it to the top
+
+        for card in cards:
+            copy_card(card, destination)
+
+
+@click.command(name='create-sprint-board')
```

Comment:
I might add an option here to control whether or not the script performs the card copy operation.

```
@click.option('--copy-cards/--no-copy-cards', default=True)
```